<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRouteDialog</class>
 <widget class="QDialog" name="OpenRouteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Serial Port</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Flow Control</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button_open_serial_port">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data Bits</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="box_flow_control"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="box_data_bits"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="box_parity"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="box_baud_rate"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="box_stop_bits"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="box_port_name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>BaudRate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Stop Bits</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port Name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="box_protocol_serial_port"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_identity_serial_port">
         <property name="text">
          <string>Identity</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="box_identity_serial_port">
         <item>
          <property name="text">
           <string>Master</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Slave</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>TCP-Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="box_tcp_server_protocol"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button_listen">
         <property name="text">
          <string>Listen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="box_tcp_server_port">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>19980</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="box_tcp_server_addr"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_identity_tcp_server">
         <property name="text">
          <string>Identity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="box_identity_tcp_server">
         <item>
          <property name="text">
           <string>Master</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Slave</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>TCP-Client</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QSpinBox" name="box_tcp_remote_server_port">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>19980</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_tcp_remote_server_addr">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="box_tcp_client_protocol"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="button_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_identity_tcp_client">
         <property name="text">
          <string>Identity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="box_identity_tcp_client">
         <item>
          <property name="text">
           <string>Master</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Slave</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>UDP</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
